doc/sphinx/language/coq-library.rst

```diff
@@ -733,7 +733,7 @@ Tactics
 
 A few tactics defined at the user level are provided in the initial
 state, in module ``Tactics.v``. They are listed at
-http://coq.inria.fr/stdlib, in paragraph ``Init``, link ``Tactics``.
+http://coq.inria.fr/stdlib/, in paragraph ``Init``, link ``Tactics``.
```

Comment:
```suggestion
https://coq.inria.fr/stdlib/, in paragraph ``Init``, link ``Tactics``.
```

Reply:
I don't mind fixing all the links in that file so that they use https. But if you want to make a lasting impact, you should start with `tools/configure/cmdArgs.ml`, as it trickles all over the place.